runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
Can you please be specific in what it is breaking? I don't understand what that commit is trying to do. I've just tried to adapt to the upstream changes as much as possible.